lib.typ

```diff
@@ -6,6 +6,9 @@
     téléphone: none,
     logo: none,
   ),
+  ecole: (
```

Comment:
I used accents for other parameters names, maybe `école` would be better to be consistent.